<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>815</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-family:微软雅黑</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="BodyLabel" name="label_3">
       <property name="text">
        <string>关键词搜索</string>
       </property>
       <property name="indent">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LineEdit" name="searchKeyWordslineEdit"/>
     </item>
     <item>
      <widget class="QCheckBox" name="proxyCheckBox">
       <property name="text">
        <string>开启代理</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="crawlSourceComboBox">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>BtHome</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ComicGarden</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="PushButton" name="searchBtn">
       <property name="text">
        <string>开始</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1102</width>
        <height>640</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="torrentPageverticalLayout">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="subtitleGroupverticalLayout">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="torrentverticalLayout">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="BodyLabel" name="label_4">
       <property name="text">
        <string>种子筛选</string>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <property name="indent">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LineEdit" name="filterlineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="BodyLabel" name="label_11">
       <property name="text">
        <string>保存位置</string>
       </property>
       <property name="indent">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LineEdit" name="savePathlineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="CheckBox" name="downloadcheckBox">
       <property name="text">
        <string>下载种子</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CheckBox" name="addTorrentcheckBox">
       <property name="text">
        <string>添加种子</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CheckBox" name="keepUpdatecheckBox">
       <property name="text">
        <string>订阅字幕组</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="PushButton" name="startBtn">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PushButton" name="updateBtn">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CheckBox</class>
   <extends>QCheckBox</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
